### DISCUSSION TITLE
Automatic processing of embargoed datasets

### DISCUSSION THREAD

**Discussion by peer35** [UtrechtUniversity/yoda#488] [Q&A] [answered]
Hi all,

We got a question about the embargo date in the Yoda metadata form. I was ready to answer that there is no functionality attached, and that it was up to the researcher or datamanager to manually publish the dataset after the embargo date.  

But in a quick search in github I found a cronjob and rule using this piece of code: https://github.com/UtrechtUniversity/yoda-ruleset/blob/50168af6f91c0d889731135445550971fe58dd4a/publication.py#L1581 (since 1.9?)

So it looks like there might actually be functionality to deal with embargo dates after all. What is the status of this and is there any documentation describing the functionality?

Best,
Peter

**Comment by lwesterhof** [comment, marked as answer]
Hi Peter,

You are correct that there is indeed functionality related to embargo dates (sonce v1.9), specifically for data packages with open access. The contents of these data packages will remain inaccessible until the specified embargo date.

While this functionality is not yet documented in the current end-user documentation, it is planned to be included in the next iteration of the documentation. 

If you have any further questions or need additional clarification, feel free to reach out.

Best regards,
Lazlo
